src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -922,216 +910,102 @@ void testErrorReportedDuringRegistrationWhenHasSoldLandFieldIsTrueForPartialVali
+    // IS SECURE REGISTER
+    // partial validation
+    @ParameterizedTest
+    @ValueSource(booleans = { false, true })
+    void testNotErrorReportedDuringRegistrationWhenIsSecureRegisterFieldIsFalseOrTrueForPartialValidationAndIsRedisRemovalEnabledFeatureFlagSetToFalse(boolean isSecureRegister) throws ServiceException {
         setIsRedisRemovalEnabledFeatureFlag(false);
         buildOverseasEntitySubmissionDto();
-        overseasEntitySubmissionDto.setHasSoldLand(Boolean.TRUE);
-
-        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(),
-                LOGGING_CONTEXT, PASS_THROUGH_HEADER);
-        assertFalse(errors.hasErrors());
-    }
-
-    void testErrorNotReportedDuringUpdateForMissingHasSoldLandFieldForFullValidation() throws ServiceException {
-        setIsRedisRemovalEnabledFeatureFlag(true);
-        setIsRoeUpdateEnabledFeatureFlag(true);
-
-        buildOverseasEntitySubmissionDto();
-        overseasEntitySubmissionDto.setEntityNumber("OE111229");
-        overseasEntitySubmissionDto.setHasSoldLand(null);
-
-        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, PASS_THROUGH_HEADER);
+        overseasEntitySubmissionDto.setIsSecureRegister(isSecureRegister);
 
+        Errors errors = overseasEntitySubmissionDtoValidator.validatePartial(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, PASS_THROUGH_HEADER);
         assertFalse(errors.hasErrors());
     }
 
     @Test
-    void testErrorReportedDuringUpdateWhenHasSoldLandFieldIsTrueForFullValidation() throws ServiceException {
-        setIsRedisRemovalEnabledFeatureFlag(true);
-        setIsRoeUpdateEnabledFeatureFlag(true);
-
-        buildOverseasEntitySubmissionDto();
-        overseasEntitySubmissionDto.setEntityNumber("OE111229");
-        overseasEntitySubmissionDto.setHasSoldLand(Boolean.TRUE);
-
-        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(),
-                LOGGING_CONTEXT, PASS_THROUGH_HEADER);
-
-        assertFalse(errors.hasErrors());
-    }
-
-    // HAS SECURE REGISTER
-    // full validation
-    @Test
-    void testNotErrorReportedDuringRegistrationWhenIsSecureRegisterFieldIsFalseForFullValidation() throws ServiceException {
-        buildOverseasEntitySubmissionDto();
-
-        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, PASS_THROUGH_HEADER);
-        assertFalse(errors.hasErrors());
-    }
-
-    @Test
-    void testNotErrorReportedDuringRegistrationWhenIsSecureRegisterFieldSetToTrueForFullValidationAndIsRedisRemovalEnabledFeatureFlagSetToFalse() throws ServiceException {
+    void testNotErrorReportedDuringRegistrationWhenIsSecureRegisterFieldSetToTrueForPartialValidationAndIsRedisRemovalEnabledFeatureFlagSetToFalse() throws ServiceException {
```

Comment:
is this test already performed by the one above (@ValueSource(booleans = { false, true }))

Reply:
you are right, forgot to remove it